js/src/personal/bryan/Bryan.module.css

```diff
@@ -21,11 +21,11 @@
 .header {
     height: rem(6vh);
     background-color: var(--mantine-color-body);
-    border-bottom: rem(1px) solid light-dark(var(--mantine-color-gray-3), var(--mantine-color-dark-4));
+    border-bottom: rem(0.2vh) solid light-dark(var(--mantine-color-gray-3), var(--mantine-color-dark-4));
```

Comment:
Really feel like vh less than 5 shouldn't really be used. I feel like vh should generally be reserved for like layouting things and not margins/padding/borders. So like 30vh or 60vh or something.